### PR TITLE
Add Kirigami QML library

### LINKED ISSUE #479: Include kirigami QML library as an option to develop apps
### Description of the feature
The Kirigami QML Library is very useful for building cross-platform QML apps that also run on most desktop distributions, Plasma Mobile and Android, not only on Ubuntu Touch. One of it's primary aims is to support convergence, which fits to Ubuntu Touch very well. It would allow to share a few apps with Plasma Mobile, for example [Kaidan](https://github.com/kaidanim/kaidan).

### Illustrations
image
App Drawer
image
An example page as displayed on a phone
image
The same page displayed on the desktop (convergence!)

### PR DESCRIPTION
**Pull request by Flohack74**
This will resolve https://github.com/ubports/ubuntu-touch/issues/479 finally

### CODE REVIEW

**Review by UniversalSuperBox — CHANGES_REQUESTED** [review]
The development packages should be added to sdk-libs-dev so that Clickable containers will pull them in.

I believe that the Kirigami package should be added to sdk-libs rather than touch since it's part of the app development kit, and could be installed on non-Touch images too.

**Review by UniversalSuperBox — APPROVED** [review]

**Comment by UniversalSuperBox** [comment]
I'll wait for a review from @bhdouglass since he's the one who will need to rebuild the Clickable containers. :)

**Comment by bhdouglass** [comment]
LGTM

**Review by bhdouglass — APPROVED** [review]

**Pull request merged**
